org.jasp.JASP.json

```diff
@@ -0,0 +1,430 @@
+{
+	"app-id": "org.jasp.JASP",
+	"runtime": "org.kde.Sdk",
+	"runtime-version": "5.10",
```

Comment:
fwiw there's a 5.11 also available now - but if you haven't tested it on there yet then I perfectly understand sticking with what works.

Reply:
Yeah, I might migrate it to there later but for now I will indeed stick with what works.